pkg/plugins/platform/local_test.go

```diff
@@ -131,6 +131,24 @@ func TestLocalStart(t *testing.T) {
 
 		cleanup()
 	}
+
+	// try to start a site that is already running
+	err = app.Init(TestSites[0].Dir)
+	assert.NoError(err)
+
+	err = app.Start()
+	assert.EqualError(err, fmt.Sprintf("a site with the name %s is already running", TestSites[0].Name))
```

Comment:
This specific message would have only been surfaced if a user tried to run start on the same exact site again.

The message you would see in the scenario you describe is here: https://github.com/drud/ddev/pull/169/files#diff-ef6770d9874ceccc0d1f5c859385f6feR56 That should indicate to the user the state of the container found and the location the found container was started from.